.editorconfig

```diff
@@ -0,0 +1,26 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+# Unix-style newlines with a newline ending every file
+end_of_line = lf
+single_quote = false
+
+# set markdown specifics
+[*.{md}]
+max_line_length = 120
+
+# docstrings and comments use max_line_length = 79
```

Comment:
is this enforced? or a preference?

Reply:
Since we are using prettier this aims to enforce it: https://prettier.io/docs/en/options.html#print-width